qemu/tests/balloon_hotplug.py

```diff
@@ -62,6 +89,9 @@ def run(test, params, env):
         if not ver_out:
             err += ("\nDevice is still in qtree %ss after unplug:\n%s"
                     % (pause, vm.monitor.info("qtree")))
+        else:
+            if pm_test_after_unplug:
+                run_pm_test(pm_test_after_unplug, "hot-unplug")
```

Comment:
Better to move this section to right behind L100, and add the following check.
```
if not vm.is_alive():
    return
```

Reply:
Modified.
Thanks